local/kaltura/version.php

```diff
@@ -25,8 +25,45 @@
     die('Direct access to this script is forbidden.');
 }
 
-$plugin->version = 2021051700;
+$plugin->version = 2021051701;
 $plugin->component = 'local_kaltura';
-$plugin->release = 'Kaltura release 4.3.0';
+$plugin->release = 'Kaltura release 4.3.1';
 $plugin->requires = 2018120300;
 $plugin->maturity = MATURITY_STABLE;
+
+global $DB;
+
+$localKalturaPluginVersionRecords = $DB->get_records_select('config_plugins', "plugin = 'local_kaltura' AND name = 'version'");
+
+$kalturaPluginVersion = "";
+if ($localKalturaPluginVersionRecords) {
+    foreach ($localKalturaPluginVersionRecords as $key => $localKalturaPluginVersionRecord) {
+        $kalturaPluginVersion = $localKalturaPluginVersionRecord->value;
+        break;
```

Comment:
this indicates you either expect only one or handle only the first and don't care about the others.
if that's the case, why foreach loop?